cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -182,12 +203,61 @@ script "js_filter_old_snapshots", type: "javascript" do
   EOS
 end
 
+script "js_get_snapshot_ami", type:"javascript" do
+  result "results"
+  parameters "snapshotId", "region"
+  code <<-EOS
+    results = {
+    "auth": "auth_aws",
+    "host": 'ec2.'+region+'.amazonaws.com',
+    "verb": "GET",
+    "path": "/",
+    "headers": {
+      "User-Agent": "RS Policies",
+      "Content-Type": "text/xml"
+    },
+    "query_params": {
+      'Action': 'DescribeImages',
+      'Version': '2016-11-15',
+      'Filter.1.Name': 'block-device-mapping.snapshot-id',
+      'Filter.1.Value.1': snapshotId
+    }
+  }
+  EOS
+end
+
+script "js_filter_ami_snapshots", type:"javascript" do
+  result "results"
+  parameters "snapshotImagesList", "snapshotsList"
+  code <<-EOS
+    var results = [];
+    var snapshotImageIds = [];
+    for(var i=0; i < snapshotsList.length; i++){
+      var snapshot = snapshotsList[i];
+      for(var j=0; j < snapshotImagesList.length; j++){
+        var snapshotImage = snapshotImagesList[j];
+        if(snapshot['snapshotId'] === snapshotImage['snapshotId']){
+          snapshotImageIds.push(snapshotImage['imageId'])
+		}
+	  }
+      results.push({
+        snapshotId : snapshot['snapshotId'],
+        region : snapshot['region'],
```

Comment:
sort by region.

Reply:
I have already sorted the list in 'js_filter_old_snapshots' block by region and daysOld